src/microbe_stage/Membrane.cs

```diff
@@ -201,6 +210,32 @@ public bool Contains(float x, float y)
         return crosses;
     }
 
+    public void EnableEngulfAnimation(bool enable, double delta)
+    {
+        if (enable && engulfFade < 1)
+        {
+            engulfFade += delta / 0.5;
```

Comment:
Divides should be rewritten as multiplies to not do expensive calculations a lot when it can be avoided. If I'm not wrong, this would just be `delta * 2` which is much easier to also read (at least to me to understand what's going on).